tests/st/runtime/test_dynamic_shape.py

```
@@ -309,53 +265,27 @@ def get_backend_type(self) -> BackendType:
 class TestDynamicShapeOperations:
     """Test suite for dynamic shape kernel operations."""
 
+    @pytest.mark.parametrize("backend", PLATFORMS)
     @pytest.mark.parametrize("shape", _SHAPES)
-    def test_dyn_shape_add(self, test_runner, shape):
+    def test_dyn_shape_add(self, test_runner, shape, backend):
         """Test add with fully dynamic M×N tensor shapes."""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)